### PR TITLE
Fixes #2050 Update R domain to accommodate Core Changes in R

### LINKED ISSUE #2050: Update R domain to accommodate Core Changes in R

### PR DESCRIPTION
**Pull request by rwmcintosh**

### CODE REVIEW

File: mobi_nuget.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
 @echo off
 cls
-rake create_local_nuget[-m]
```
Comment (rwmcintosh) [inline mobi_nuget.bat:3]: these dashes aren't necessary I remember

File: src/OSPSuite.Core/Domain/Builder/SimulationBuilder.cs [hunk 2]
```diff
@@ -62,27 +62,27 @@ internal IEnumerable<InitialCondition> AllPresentMoleculeValuesFor(IEnumerable<s
             .Where(initialCondition => initialCondition.IsPresent);
       }
 
-      internal IEnumerable<MoleculeBuilder> AllFloatingMolecules() => Molecules.Where(x => x.IsFloating);
```
Comment (rwmcintosh) [inline src/OSPSuite.Core/Domain/Builder/SimulationBuilder.cs:65]: hmm, I'm rethinking that. Public only for the string lists.

File: src/OSPSuite.R/Domain/Simulation.cs [hunk 3]
```diff
@@ -47,8 +47,11 @@ public void AddAnalysis(ISimulationAnalysis simulationAnalysis)
       public Simulation(IModelCoreSimulation modelCoreSimulation)
       {
          CoreSimulation = modelCoreSimulation;
+         BuildConfiguration = new SimulationBuilder(Configuration);
       }
 
+      public SimulationBuilder BuildConfiguration { get; set; }
```
Comment (msevestre) [inline src/OSPSuite.R/Domain/Simulation.cs:53]: not set

**Review by msevestre — CHANGES_REQUESTED** [review]
Remove public setter. This should never be set from the outside

**Pull request merged**